.gitattributes

```diff
@@ -1,2 +0,0 @@
-/.circleci/config/@build-release.yml
-/packagespec.mk linguist-generated
```

Comment:
Is this related to CircleCI or is this just something we don't use anymore?

Reply:
Something we havn't used in a while.